### PR TITLE
feat: profile, individual Profile, follow/unfollow

### PR DESCRIPTION
**Pull request by vikrantkode**
Profile Page of User
Profile Page Of Individual User
Follow/Unfollow User
STatic comments

### CODE REVIEW

**Review by sharmayatendra — APPROVED** [review]
looks good to merge

File: src/Pages/Auth/authSlice.js [hunk 1]
```diff
@@ -27,60 +64,116 @@ export const logInUser = createAsyncThunk(
...
 
 )
 
-
+export const updateUserProfile = createAsyncThunk(
+    `profile/updateUserProfile`,
+    async ({userData}, { rejectWithValue }) => {
+        try {
+            const resp = await updateUserProfileService(userData)
+            console.log(resp)
```
Comment (tushargoyal812) [inline src/Pages/Auth/authSlice.js:89]: remove console

File: src/Pages/Profile/OtherUserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    {individualUser?.firstName} {individualUser?.lastName}
+                  </p>
+                  <p className="ml-4 text-sm text-slate-400 ">
+                    @{individualUser?.userHandle}
+                  </p>
+                </div>
+                {/* <button className="rounded-md font-semibold text-gray-500 hover:bg-blue-400 mr-4 px-4 m-4"
+                >
+                  Edit
+                </button> */}
```
Comment (tushargoyal812) [inline src/Pages/Profile/OtherUserProfile.jsx:39]: remove comments

File: src/Pages/Profile/ProfileModal.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { toast } from "react-toastify";
+import { updateUserProfile } from "../Auth/authSlice";
+
+const ProfileModal = ({setEditProfileModalVisibility}) => {
+  const {user} = useSelector((state) => state.auth)
+  console.log(user)
```
Comment (tushargoyal812) [inline src/Pages/Profile/ProfileModal.jsx:8]: remove console

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge just few changes needed

**Comment by vikrantkode** [comment]
Thanks for reviewing

**Pull request merged**